### ISSUE TITLE
Distributed PourOver

### ISSUE THREAD

**Issue by hhsnopek** [NYTimes/pourover#56]
I understand that everyone is busy with work and other problems that have a higher priority over PourOver. I recently sent an email to @esmooov and @bkoski asking to add myself as a maintainer of PourOver. Without any response over the past week I have gone ahead and [forked PourOver](//github.com/hhsnopek/pourover) and published it with the latest changes from #47. This fork will continue to grow and improve pourover. If anyone would like to help maintain PourOver I'd be glad to grant them access to the repo.

**Comment by tiffehr** [comment]
Added a note and link to your version to our description of the project.  Cool to close this Issue, with that pointer in place?

**Comment by hhsnopek** [comment]
Sure it would be if you checked over my fork and possibly merged or cherry pick some of the commits out.

Improvements:
- dist folder, separate from source
- travis-ci for continous integration and testing
- bower/component/npm that include UMD/AMD/Global support
- build process using gulp
- tests refactored and faster using mocha

Upstream improvements:
- Source refactor and separation of the library, similar to how moment.js has split everything
- es6 support
- benchmarking
- updated github page

**Comment by hhsnopek** [comment]
Oh and!
improvements cont:
- snack_case to CamelCase to follow javascript standards
- deprecations

**Comment by hhsnopek** [comment]
Actually now that I think of it, it could be good to just base the next version off of [Pourover v1.2.0](https://github.com/hhsnopek/pourover/releases/tag/v1.2.0)

**Comment by tiffehr** [comment]
To be frank, what's the main reason to support a next version?  Use within NYT is very limited—I inherited our main project using it and its own lifespan is not forever.  It was meant more to be of immediate use on a few projects, as well as inspiration and discussion for people looking at similar problems.

NYT can't and shouldn't make it solve every other use-case or dev-env need.  If it's not our own, it's just blind adherence.

**Comment by hhsnopek** [comment]
Main reason to support new versions are for improvements beyond what's been made, creating ease for the community that uses the project. The majority of changes made and requested by myself and others are helping PourOver stay up to date with the latest build systems, package managers, and overall javascript. You don't have to support every developers request, but as of my release of v1.1.1, I gave the community what they needed to continue to use PourOver with the newest libraries. While also making it easier for developers to continue to build off of this wonderful library.

You're right NYT can't and shouldn't make it solve every other use-case or dev-env need, but a lot of those are already completed in v1.1.1. After that release everything is to make the project more developer friendly. If not basing off of v1.2.0, at least v1.1.1 as it brings structure, continuos integration, a build process for source/minified versions, and better performance with tests.

**Comment by tiffehr** [comment]
We spent some time discussing this today.  We'll make some of these changes, for sure.  Yours is definitely the better home for a lot of the improvements you want to see.  We'll add some clarifications and pointers to your repo, for those who want more than our basic needs.

Expect more on that soon.  Taking up this repo again isn't tied to any current work on our part, so it'll happen around those deadlines.  We found a bug within a recent project and that triggered changing up repo ownership.
